lib/ci/docker.rb

```diff
@@ -0,0 +1,5 @@
+require 'docker'
+
+if Gem::Version.new(Docker::API_VERSION) < Gem::Version.new(1.24)
+  Docker::API_VERSION = '1.24'.freeze
```

Comment:
I just noticed. this coerces the version regardless of whether the server actually supports that version. it takes no measures if that is not the case